Sources/MessagingInApp/Gist/Managers/MessageManager.swift

```diff
@@ -19,7 +19,7 @@ class MessageManager {
     var engine: EngineWebInstance
     private let siteId: String
     let currentMessage: Message
-    var gistView: GistView!
+    let gistView: GistView
```

Comment:
This PR adds code that removes the WebView from the UI and performs resource cleanup. Using `!` adds risk to a runtime crash and I thought there would be a possibility in the future where someone tries to use `gistView = nil` and cause the SDK to crash. To prevent this, I refactored `gistView` to be immutable.